undebt/pattern/interface.py

```diff
@@ -78,10 +84,23 @@ def patterns_from_files(pattern_files):
 
 def load_module(path):
     """Loads a module from its path."""
+    if module_like(path):
+        path = module_name_to_path(path)
```

Comment:
Hmm, from the docs:

| This function does not handle hierarchical module names (names containing dots). In order to find P.M, that is, submodule M of package P, use find_module() and load_module() to find and load package P, and then use find_module() with the path argument set to P.__path__. When P itself has a dotted name, apply this recipe recursively.

Seems like this could get potentially gross.